laget.Azure.ServiceBus/Topic/TopicReceiver.cs

```diff
@@ -1,32 +1,89 @@
-﻿using System;
-using System.Threading;
-using System.Threading.Tasks;
+﻿using Azure.Storage.Blobs;
 using Microsoft.Azure.ServiceBus;
 using Microsoft.Azure.ServiceBus.Core;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
 
 namespace laget.Azure.ServiceBus.Topic
 {
+    using Message = Microsoft.Azure.ServiceBus.Message;
+
     public interface ITopicReceiver
     {
-        void Register(Func<Microsoft.Azure.ServiceBus.Message, CancellationToken, Task> callback, Func<ExceptionReceivedEventArgs, Task> exceptionHandler, MessageHandlerOptions handlerOptions = null);
+        void Register(Func<Message, CancellationToken, Task> callback, Func<ExceptionReceivedEventArgs, Task> exceptionHandler, MessageHandlerOptions handlerOptions = null);
     }
 
     public class TopicReceiver : ITopicReceiver
     {
         private readonly IMessageReceiver _client;
+        private readonly BlobContainerClient _blobContainerClient;
+        private readonly string _topic;
+
+        public TopicReceiver(string connectionString, TopicOptions options) :
```

Comment:
Vi brukar generellt sätta `: this(...)` och `: base(...)` på en ny rad, alltså att `:` ligger med på raden under.